thamos/lib.py

```diff
@@ -48,7 +48,6 @@
 from .swagger_client import BuildAnalysisApi
 from .swagger_client import Configuration
 from .swagger_client import PythonStack
-from .swagger_client import RuntimeEnvironment
 from .swagger_client import AdviseInput
```

Comment:
do we remove it because we check from common config now?

Reply:
We don't want to have fixed runtime environment entries as provided by the API model. In that case, we will introduce a tight coupling between thamos and user-api versions. Users will need to make sure thamos version is the right for communicating with thoth backend (exact match), which is not nice. With this change, the API schema is not strictly enforced, but users will get warning messages if they use an older or newer thamos version when communicating with thoth (can be seen logs).

Reply:
Thanks for exaplanation :)